api/src/pcapi/routes/backoffice/chronicles/forms.py

```diff
@@ -53,6 +55,15 @@ class Meta:
         choices=[("true", "Oui"), ("false", "Non")],
     )
 
+    category = fields.PCSelectMultipleField(
+        "Catégorie d’oeuvre",
```

Comment:
```suggestion
        "Club",
```

ça ne se rapporte pas au produit mais au club, je pense que mettre type d'oeuvre est trompeur.